### PR TITLE
fix go-realtime-gui.bat

### PR DESCRIPTION
**Pull request by azure-crab**
- add packages gui_v1.py requires to run:
  - PySimpleGUI
  - noisereduce
- rewrite go-realtime-gui.bat, add comments and some configuration.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -43,3 +43,6 @@ onnxruntime-gpu; sys_platform != 'darwin'
 torchcrepe==0.0.20
 fastapi==0.88
 ffmpy==0.3.1
+#下面是使用go-realtime-gui.bat需要依赖的包
```
Comment (Tps-F) [inline requirements.txt:46]: There is no need to write it here because it is in requirements-win-for-realtime_vc_gui.txt!

**Review by Tps-F — CHANGES_REQUESTED** [review]

**Comment by Tps-F** [comment]
https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/issues/667#issuecomment-1616350969
As he said, this is the file for https://huggingface.co/lj1995/VoiceConversionWebUI/tree/main
so you may not need to change it.

File: requirements.txt (hunk 1 above)
Comment (fumiama) [reply requirements.txt:46]: Exactly.

File: go-realtime-gui.bat [hunk 2]
```diff
@@ -1,2 +1,12 @@
-runtime\python.exe gui_v1.py
-pause
+@echo off
+
+@REM 定义你使用的python，指向你配好的环境中的python路径
+@REM     如 C:\Users\yourname\miniconda3\envs\rvc\python.exe
+set PYTHON=
+
+@REM 默认配置
+if not defined PYTHON (set PYTHON=python)
```
Comment (fumiama) [inline go-realtime-gui.bat:8]: 默认应当保持runtime\python.exe

**Review by fumiama — CHANGES_REQUESTED** [review]

**Pull request closed without merging**